src/components/spreadsheet/equipment-table.js

```diff
@@ -95,8 +107,11 @@ export const EquipmentTable = ({
             wrapHeaderText: true,
             autoHeaderHeight: true,
             suppressKeyboardEvent: (params) => suppressKeyEvent(params),
+            icons: {
+                menu: filterIcon,
+            },
```

Comment:
i think we can directly set :
icons: {
                menu: '<span class="ag-icon ag-icon-filter"/>',
            },
and remove filterIcon const

Reply:
Yes it seems a bit overkill, i removed it